.github/workflows/deploy.yml

```diff
@@ -32,6 +37,8 @@ jobs:
       id-token: write
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.version }}
```

Comment:
Similar to previous comment, rather than checking out inputs.version, we want instead to pass in IMAGE_TAG to `make release-deploy`